corehq/project_limits/tests/test_system_limit.py

```diff
@@ -23,7 +23,9 @@ def test_for_key_returns_domain_specific_limit(self):
         self.assertEqual(SystemLimit.for_key("general_limit"), 10)
         self.assertEqual(SystemLimit.for_key("general_limit", domain="specific"), 20)
 
-    def test_cache_is_cleared(self):
+    def test_patching_cache_works(self):
+        # if the logic in ../patch_cache.py fails, this test should catch that
```

Comment:
nit: `patch_cache.py` is in the same directory as this test module, correct?
```suggestion
        # if the logic in patch_cache.py fails, this test should catch that
```

Reply:
a091831ffba8c94800afe0b2a1be93bd7fd41171